### ISSUE TITLE
[IQE] : [Ctrl+PageUp/PageDown] shortcuts on Navigate menu are not localized

### ISSUE THREAD

**Issue by rawat11** [adobe/brackets#11336]
Steps To Reproduce:

1) Launch Brackets
2) Open and edit some files in the Working set
3) Go to Navigate menu

Result:  Next Document (List Order) and Previous Document (List Order) are not localized 

ENV : Release 1.4 build 1.4.0-16190 (master e35f2a4) (MAC and WIN)
![screen shot 2015-07-02 at 4 55 57 am](https://cloud.githubusercontent.com/assets/10069885/8476568/2a425eac-20e0-11e5-9892-40f939348b26.png)

**Comment by prafulVaishnav** [comment]
@rawat11 Hey.. We are not getting ALF automation update for string changes. Last update we got was on 22 April. Can you please look into it.

**Comment by rawat11** [comment]
@prafulVaishnav Yes I have  recieved the strings for it, we will send it for translation
